CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
 
+### Breaking Change
+- bumped dependency of `sensu-plugin` to 2.x (@majormoses)
```

Comment:
what do you think about linking to the `sensu-plugin` 2.0 changelog (https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#v200---2017-03-29). I think that would go a long way to help people understand what has changed with 2.x and take action if needed.

Reply:
they also leave off a very important one which I guess we could add. They depend on json > 2 as well...

Reply:
looks like it was added post 2.x as a patch (but not called out as a breaking change). Should we update that to call it a Breaking Change.

Reply:
Good question. Technically we're fine because the "drop support for ruby 1.9 and earlier" entry covers the breaking change in json 2 (https://github.com/flori/json/blob/master/CHANGES.md#2015-09-11-200)

> Drops support for old rubies whose life has ended, that is rubies < 2.0. Also see https://www.ruby-lang.org/en/news/2014/07/01/eol-for-1-8-7-and-1-9-2/

Reply:
We should add a link to the [blog talking about filtering changes](https://blog.sensuapp.org/deprecating-event-filtering-in-sensu-plugin-b60c7c500be3). It's linked in the sensu-plugin 1.4.0 entry but nobody is going to look at that when looking at 2.0 changes.

Reply:
Ya but someone needs to follow another changelog (which it apparently was not obvious when it was released) so I would imagine other users even less likely to find that.

Reply:
> 1.4.0 entry but nobody is going to look at that when looking at 2.0 changes.

Then it should be relinked in 2.x changelog entry. we are gonna have to do this almost 200 times I dont want to have to write the same thing out a bunch of times. That's the point of a CHANGELOG is so that we don't have to. I'd say we touch up `sensu-plugin` 2.x changelog entry and then link to that and call it good.

Reply:
Yup, that's what I was suggesting. We are on the same page.